http/vertx/src/test/resources/server.crt

```diff
@@ -0,0 +1,29 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
No :-) You are adding some hardcoded certificates without any information how you created it and I have no idea if it works in FIPS. Please configure HTTPS using FW. It should be enough to do something like `@QuarkusApplication(ssl = true, certificates = @Certificate(format = PEM, configureHttpServer = true, configureKeystore = true, configureTruststore = true))` and you can drop 

```
.withProperty("quarkus.http.ssl.certificate.files", "server.crt")
.withProperty("quarkus.http.ssl.certificate.key-files", "server.key");
````

You can also set `useTlsRegistry = false` if you insist on avoiding TLS registry but I expect it is irrelevant for your test so don't do it.

Reply:
Ok thanks @michalvavrik , I will change to use FW.